JetNews/app/src/main/java/com/example/jetnews/ui/home/HomeScreen.kt

```diff
@@ -166,15 +170,37 @@ fun HomeScreen(
         topBar = {
             val title = stringResource(id = R.string.app_name)
             InsetAwareTopAppBar(
-                title = { Text(text = title) },
+                title = {
+                    Image(
+                        painter = painterResource(R.drawable.ic_jetnews_wordmark),
+                        contentDescription = title,
+                        colorFilter = ColorFilter.tint(MaterialTheme.colors.onBackground),
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .padding(bottom = 4.dp, top = 10.dp)
+                    )
+                },
                 navigationIcon = {
                     IconButton(onClick = { coroutineScope.launch { openDrawer() } }) {
-                        Icon(
+                        Image(
                             painter = painterResource(R.drawable.ic_jetnews_logo),
-                            contentDescription = stringResource(R.string.cd_open_navigation_drawer)
+                            contentDescription = stringResource(R.string.cd_open_navigation_drawer),
+                            colorFilter = ColorFilter.tint(MaterialTheme.colors.primary)
+                        )
+                    }
+                },
+                actions = {
+                    IconButton(
+                        onClick = { /* TODO: Open search */ }
+                    ) {
+                        Icon(
+                            imageVector = Icons.Filled.Search,
+                            contentDescription = stringResource(R.string.cd_search)
                         )
                     }
-                }
+                },
+                backgroundColor = MaterialTheme.colors.onPrimary,
+                elevation = 0.dp
```

Comment:
Same comment about elevation and background color

Reply:
Fixed it

Reply:
Added elevation when scrolling